src/modules/home.js

```diff
@@ -0,0 +1,58 @@
+document.addEventListener('DOMContentLoaded', () => {
+  const mealList = document.querySelector('.meal-list');
+
+  const createMealDiv = (thumbnail, name, recipe) => {
+    const mealDiv = document.createElement('div');
+    mealDiv.className = 'meal';
+
+    const thumbnailElement = document.createElement('img');
+    thumbnailElement.src = thumbnail;
+    thumbnailElement.alt = name;
+
+    const nameElement = document.createElement('h2');
+    nameElement.textContent = name;
+
+    const recipeElement = document.createElement('p');
+    recipeElement.textContent = recipe;
+
+    mealDiv.append(thumbnailElement, nameElement, recipeElement);
+    return mealDiv;
+  };
+
+  const createButton = (text, className) => {
+    const button = document.createElement('button');
+    button.textContent = text;
+    button.className = className;
+    return button;
+  };
+
+  fetch('https://www.themealdb.com/api/json/v1/1/filter.php?c=Chicken')
+  //   fetch('www.themealdb.com/api/json/v1/1/random.php')
```

Comment:
- [ ] To make your code really clean, could you remove this old piece of code since it's not in use? It reduces chances of bugs, or lack of clarity. :wink: